### ISSUE TITLE
Feature Request - Risk

### ISSUE THREAD

**Issue by ja2cool** [ninjaspread/ns#6]
Interested in a filter that would limit results based on a price range for the trade's max risk. This would apply to credit trades.

**Comment by ninjaspread** [comment]
Do you mean another filter besides price that has a min and max of risk?

**Comment by ja2cool** [comment]
Yes, min/max risk. I know for debit trades it's obvious but for credit
spreads and iron condors it would be helpful.
